examples/CMakeLists.txt

```diff
@@ -9,6 +9,7 @@ list(APPEND _examples
     mpi_shared_tree
     mpi_send_tree
     mpi_matrix
+	schrodinger_semigroup1d
```

Comment:
Indentation

Reply:
This depends on the editor one uses to browse the file. In Visual Studio the indentation is fine, and I do not know how to make it the same in all editors.

Reply:
Well it should not. I guess the settings og your editor should be twaked :-)

Reply:
Seems you're using 2 tabs instead of one.

Reply:
I used firstly one tab in a normal text editor, which turned out to be different from the indention used. However, when I opened it in vs code, it was fine. So my guess is that the others are done in vs code by using space key. Yes, it seems that Stig used spaces before, and I never use space key for indentation.

Reply:
Done